### ISSUE TITLE
cijoe -v blows out

### ISSUE THREAD

**Issue by millisami** [defunkt/cijoe#30]
```
ree-1.8.7-2010.02 [~] ➔ gem list cijoe

*** LOCAL GEMS ***

cijoe (0.7.0)
ree-1.8.7-2010.02 [~] ➔ cijoe -v
/Users/millisami/.rvm/gems/ree-1.8.7-2010.02/gems/cijoe-0.7.0/bin/cijoe:41: uninitialized constant CIJoe (NameError)
    from /Users/millisami/.rvm/gems/ree-1.8.7-2010.02/gems/choice-0.1.4/lib/choice/parser.rb:177:in `call'
    from /Users/millisami/.rvm/gems/ree-1.8.7-2010.02/gems/choice-0.1.4/lib/choice/parser.rb:177:in `parse'
    from /Users/millisami/.rvm/gems/ree-1.8.7-2010.02/gems/choice-0.1.4/lib/choice/parser.rb:155:in `each'
    from /Users/millisami/.rvm/gems/ree-1.8.7-2010.02/gems/choice-0.1.4/lib/choice/parser.rb:155:in `parse'
    from /Users/millisami/.rvm/gems/ree-1.8.7-2010.02/gems/choice-0.1.4/lib/choice.rb:93:in `parse'
    from /Users/millisami/.rvm/gems/ree-1.8.7-2010.02/gems/choice-0.1.4/lib/choice.rb:26:in `options'
    from /Users/millisami/.rvm/gems/ree-1.8.7-2010.02/gems/cijoe-0.7.0/bin/cijoe:6
    from /Users/millisami/.rvm/gems/ree-1.8.7-2010.02/bin/cijoe:19:in `load'
    from /Users/millisami/.rvm/gems/ree-1.8.7-2010.02/bin/cijoe:19
ree-1.8.7-2010.02 [~] ➔ 
```

**Comment by matthewford** [comment]
On 1.9.2 too:

/home/ubuntu/.rvm/gems/ruby-1.9.2-p0/gems/choice-0.1.4/lib/choice/parser.rb:108: warning: Hash#index is deprecated; use Hash#key
/home/ubuntu/.rvm/gems/ruby-1.9.2-p0/gems/cijoe-0.7.0/bin/cijoe:41:in `block (3 levels) in <top (required)>': uninitialized constant Choice::Option::CIJoe (NameError)
    from /home/ubuntu/.rvm/gems/ruby-1.9.2-p0/gems/choice-0.1.4/lib/choice/parser.rb:177:in`call'
    from /home/ubuntu/.rvm/gems/ruby-1.9.2-p0/gems/choice-0.1.4/lib/choice/parser.rb:177:in `block in parse'
    from /home/ubuntu/.rvm/gems/ruby-1.9.2-p0/gems/choice-0.1.4/lib/choice/parser.rb:155:in`each'
    from /home/ubuntu/.rvm/gems/ruby-1.9.2-p0/gems/choice-0.1.4/lib/choice/parser.rb:155:in `parse'
    from /home/ubuntu/.rvm/gems/ruby-1.9.2-p0/gems/choice-0.1.4/lib/choice.rb:93:in`parse'
    from /home/ubuntu/.rvm/gems/ruby-1.9.2-p0/gems/choice-0.1.4/lib/choice.rb:26:in `options'
    from /home/ubuntu/.rvm/gems/ruby-1.9.2-p0/gems/cijoe-0.7.0/bin/cijoe:6:in`<top (required)>'
    from /home/ubuntu/.rvm/gems/ruby-1.9.2-p0/bin/cijoe:19:in `load'
    from /home/ubuntu/.rvm/gems/ruby-1.9.2-p0/bin/cijoe:19:in`<main>'

**Comment by defunkt** [comment]
Fix require order in ./bin/cijoe. Closed by 70ab1d5bb182dabaf3eb55c7a984c2f37f34775d
